README.md

```diff
@@ -1,2 +1,17 @@
 # HW01
-Edit the readme
+ 
+Hi everyone,
+
+I am **Xing Yanfeng** from China. I was born and grew up in *Shanxi*, a province in central China, and I just graduated from Tsinghua University. I worked in [Zhu Lab](http://zhulab.life.tsinghua.edu.cn/) for two years at Tsinghua. Our group focuses on mirror-image biology. 
+
+![Mirror-image biology](http://zhulab.life.tsinghua.edu.cn/wp-content/uploads/2012/06/Cover2NoText1.jpg)
```

Comment:
I like the picture - I am wondering if there is a way to control the size of the image? I had a similar issue where the picture I added was as big as the entire page, so I removed it. I think a smaller image size would make the document more readable.

Reply:
Good question! I just tried and I can make the image smaller by this syntax
![caption](path to image){width=50%}
It works well. Thank you for your question!